microfrontends/app-builder-menu/src/utils/keycloak.ts

```diff
@@ -0,0 +1,3 @@
+export const getKeycloakTokenFromWindow = (): string => {
```

Comment:
I would call it simply `getKeycloakToken` because window is an implementation detail of the function.